CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+### backend-metal-0.8.2 (08-05-2021)
```

Comment:
Seems not come after https://github.com/blaind/wgpu/commit/d90b30e9c4e29578a87b5968c92e95b170a530ff, not entirely sure what caused it... will try to fix.

Reply:
Looks like the PR is still off the base

Reply:
could you try rebasing properly on master?

Reply:
I branched off from hal-0.8 (specifically commit e625940050cd184d8696a967f0b227936674edfa, 0.8.2). Had to do that because otherwise I had hard-to-debug problems with syncing gfx/wgpu/wgpu-rs to compatible versions.

Seems like if I change the gfx base to hal-0.8, those commits aren't visible here. 

However, the underlying code is used by https://github.com/blaind/xrbevy and people are actively testing, I may not be able to base the gfx/wgpu/wgpu-rs all to latest commits... 

I guess one option would be to branch the pull request from the `bevy_openxr` branch to a separate pull request branch, and update the latest commits when the `bevy_openxr` branch develops further.

Especially now that people are testing, some changes are to be expected.

Furthermore, I do not know if this pull request is eventually the best approach to land the code.

Reply:
I see. This code should definitely target gfx master, not 0.8 branch.
> Had to do that because otherwise I had hard-to-debug problems with syncing gfx/wgpu/wgpu-rs to compatible versions.

Let us help with this. We override gfx stuff locally all the time. There are even commented out sections of this override in Cargo toml for both wgpu and wgpu-rs.